src/components/panel/demoBasicUsage/script.js

```diff
@@ -0,0 +1,33 @@
+angular.module('panelDemo', ['ngMaterial'])
+    .controller('BasicDemoCtrl', BasicDemoCtrl)
+    .controller('DialogCtrl', DialogCtrl);
+
+
+function BasicDemoCtrl($mdPanel) {
+  this._mdPanel = $mdPanel;
+}
+
+
+BasicDemoCtrl.prototype.showDialog = function() {
```

Comment:
I was thinking showDialog because the specific example is for dialogs. I intend to add more demos for other uses of the panel, such as a menu.

Using showDialog will allow me to have a different function for these other examples, but if you think it will lead to confusion, I can call it something else.